pkg/kubernetes/client.js

```diff
@@ -709,8 +754,13 @@ define([
          */
         self.include = function include(type) {
             var endpoint = TYPE_APIS[type] || API_KUBE;
+            var namespace = null;
+            if (selected_namespace && type_is_namespaced(type))
+                namespace = selected_namespace;
+
             if (!self.watches[type]) {
                 self.watches[type] = new KubernetesWatch(type, endpoint,
+                                                         namespace,
```

Comment:
There seems to be two ways to set the namespace on a watch, one when it's created, and another later via change_namespace(). Should we just have one? It would mean an extra function call here, but it seems more straight forward. Or is there a reason for this?